packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import { Icon, Popover } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+
+const TradersHubOnboarding = observer(() => {
+    const history = useHistory();
+    const { traders_hub, ui } = useStore();
+    const { setIsOnboardingVisited } = traders_hub;
+    const { is_dark_mode_on } = ui;
+
+    return (
+        <div data-testid='dt_traders_hub_onboarding'>
+            <div className='traders-hub-header__tradershub--onboarding--logo'>
+                <Popover
+                    classNameBubble='account-settings-toggle__tooltip'
+                    alignment='bottom'
+                    message={<Localize i18n_default_text='View onboarding' />}
+                    should_disable_pointer_events
+                    zIndex='9999'
+                >
+                    <Icon
+                        data_testid='dt_traders_hub_onboarding_icon'
```

Comment:
can you refrain of using data_testid?
it should be the last option in the selector choice for the test cases

Reply:
it renders a svg icon on screen, and since the Icon component takes data-testid prop, i passed one for testing.